lang/rust/rust.py

```diff
@@ -0,0 +1,217 @@
+from talon import Module, Context, actions, ui, imgui, settings
+
+mod = Module()
+
+ctx = Context()
+ctx.matches = r'''
+mode: user.rust
```

Comment:
We've migrated to just using a tag for these matches as part of #776. Please update when you have a moment.

see https://github.com/knausj85/knausj_talon/blob/2d9f4afd73c2566fd5d3bc4a89251c746245cdbe/lang/csharp/csharp.py#L5